curiosrc/alertmanager/task_alert.go

```diff
@@ -0,0 +1,395 @@
+package alertmanager
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	logging "github.com/ipfs/go-log/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/lib/harmony/harmonydb"
+	"github.com/filecoin-project/lotus/lib/harmony/harmonytask"
+	"github.com/filecoin-project/lotus/lib/harmony/resources"
+	"github.com/filecoin-project/lotus/node/config"
+	"github.com/filecoin-project/lotus/storage/ctladdr"
+)
+
+var MinimumBalance = types.MustParseFIL("5 FIL")
```

Comment:
How about 0.01 FIL? 
When it's at $60/FIL, it's going to be hard to meet 5 in all the different wallets.

Reply:
I was actually thinking about making this a config param so users can adjust based on their setup. it cover what you described as scenarios where I am not sealing and just proving so I shouldn't need to keep sealing wallets full.